### PR TITLE
Added standards to files

### PR DESCRIPTION
**Pull request by milesflo**
Also added Questions to one Insight missing them.

*Points were not added at this time. *

### CODE REVIEW

**Comment by enki-hq** [comment]
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/intro-js/console-in-javascript.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/intro-js/using-javascript-on-a-web-page.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-i/avoid-using-with.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-i/be-careful-when-using-typeof.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-i/catch-is-block-scoped.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-i/changing-a-functions-context-with-fn-call-object.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-i/check-if-a-function-exists.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-i/easily-generate-a-random-hex-color.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-i/get-a-random-item-from-an-array.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-i/setting-default-values-with-short-circuiting.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-i/use-the-apply-function-to-get-the-min-or-max-of-an-array.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-i/using-test-with-regexp.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-ii/counting-the-months-from-zero.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-ii/format-currency-in-javascript.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-ii/milliseconds-since-epoch.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-ii/performing-a-function-at-timed-intervals.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-ii/round-numbers-to-n-decimals.md?ref=3a0cefc8dd1557a265345702b132dcb2bc3377bd

File: JavaScript/Core/quirks-i/changing-a-functions-context-with-fn-call-object.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ type: normal
 
 category: must-know
 
+standard: js.evaluate-expressions
```
Comment (lizTheDeveloper) [inline JavaScript/Core/quirks-i/changing-a-functions-context-with-fn-call-object.md]: I added a standard for this one - https://github.com/sagelabs/standards/blob/master/javascript/core/use-and-manipulate-the-execution-context-of-a-function-in-javascript.md

File: JavaScript/Core/recipes-i/get-a-random-item-from-an-array.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ type: normal
 
 category: how to
 
+standard: js.write-expressions
```
Comment (lizTheDeveloper) [inline JavaScript/Core/recipes-i/get-a-random-item-from-an-array.md]: data types and structures for this one

File: JavaScript/Core/recipes-ii/performing-a-function-at-timed-intervals.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ type: normal
 
 category: how to
 
+standard: js.write-expressions
```
Comment (lizTheDeveloper) [inline JavaScript/Core/recipes-ii/performing-a-function-at-timed-intervals.md]: perform and manage async

**Comment by lizTheDeveloper** [comment]
I merged this into the js-standards branch already

**Pull request closed without merging**